canonical-websites/snapcraft.io
# Enable prometheus metrics for flask requests

Fixes #338

(And update ./run script)

QA
--

``` bash
./run build
./run test
./run
``` bash

Browse to http://127.0.0.1:8004, check the site works.

Now kill the server and run this instead:

``` bash
./run exec -p 8004 -p 9990 -e FLASK_APP=app.py -e FLASK_DEBUG=false flask run -h 0.0.0.0 -p 8004
```

Browse to http://127.0.0.1:8004, click around a bit. Then go to http://127.0.0.1:9990 and see the prometheus metrics for the requests you just made.
